tests/kuttl/tests/ovn_readiness/02-patch.yaml

```diff
@@ -0,0 +1,13 @@
+#
+# Step for:
+#
+# - Reduce the failureThreshold and periodSeconds to detect a failure in OVN SB DB before it reconciles
+#
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+    - script: |
+        oc patch daemonset ovn-controller -n $NAMESPACE --type='json' -p='[
```

Comment:
any changes to daemonset will not persist, but could trigger container restart, but seems may have issues some time.
what about doing it differently and avoid this steps and sb pod delete and ready false checks below?
- check connection status and it should be connected
- configure ovn-remote to some invalid address
- check connection-status and it should be not != connected